tmd/view/plot.py

```diff
@@ -34,10 +34,13 @@ def barcode(ph, new_fig=True, subplot=False, color="b", linewidth=1.2, **kwargs)
     ph_sort = analysis.sort_ph(ph)
 
     for ip, p in enumerate(ph_sort):
```

Comment:
Please use informative names to make code more readable, such as index and bar for example.